eng/azure-pipelines.yml

```
@@ -101,6 +101,27 @@ stages:
           parameters:
             enablePublishTestResults: true
 
+      - ${{ if eq(variables.officialBuild, 'false') }}:
+        - job: Bootstrap
+          condition: eq(variables.officialBuild, 'false')
+          pool:
+            name: Hosted Ubuntu 1604
```

do you really need Ubuntu 16.04? AzDO throws up warnings now that it will be removed soon: https://github.com/actions/virtual-environments/issues/3287

```suggestion
            vmImage: ubuntu-20.04
```